app/src/test/java/org/mozilla/fenix/home/intent/FennecPinnedShortcutsIntentProcessorTest.kt

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.intent
+
+import android.content.Intent
+import android.net.Uri
+import assertk.assertAll
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isTrue
+import io.mockk.Called
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.verify
+import io.mockk.verifyAll
+import kotlinx.coroutines.runBlocking
+import mozilla.components.browser.session.Session
+import mozilla.components.browser.session.SessionManager
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.feature.intent.ext.getSessionId
+import mozilla.components.feature.session.SessionUseCases
+import org.junit.Assert
+import org.mozilla.fenix.home.intent.FennecPinnedShortcutsIntentProcessor.Companion.ACTION_FENNEC_HOMESCREEN_SHORTCUT
+import org.junit.Assert.assertFalse
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.TestApplication
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import java.util.UUID
+
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+class FennecPinnedShortcutsIntentProcessorTest {
+    private val sessionManager = mockk<SessionManager>(relaxed = true)
+    private val loadUrlUseCase = mockk<SessionUseCases.DefaultLoadUrlUseCase>(relaxed = true)
+
+    @Test
+    fun `only match Fennec pinned shortcut Intents`() {
+        val processor = FennecPinnedShortcutsIntentProcessor(sessionManager, loadUrlUseCase)
+
+        Assert.assertTrue(processor.matches(Intent(ACTION_FENNEC_HOMESCREEN_SHORTCUT)))
```

Comment:
nit: 
```suggestion
        assertTrue(processor.matches(Intent(ACTION_FENNEC_HOMESCREEN_SHORTCUT)))
```

Reply:
Thanks.
There was a mix of junit and assertk assertions used.
Cleaned up the code to only use asserk ones.